packages/react-component-library/src/components/RangeSlider/Slider.tsx

```diff
@@ -89,7 +89,7 @@ const StyledRangeSlider = styled.div<StyledRangeSliderProps>`
     `}
 `
 
-const StyledSlider = styled(Slider)`
+const StyledSlider = styled<React.ComponentType<SliderProps>>(Slider)`
```

Comment:
Nice I had been wondering how to type this situation in a few other place where I've ended up with an `any`.